scripts/configure.sh

```diff
@@ -74,6 +74,10 @@ set_keybindings() {
     dconf write ${KEYS_GNOME_WM}/switch-to-workspace-down "['<Primary><Super>Down','<Primary><Super>${down}']"
     # Move to workspace above
     dconf write ${KEYS_GNOME_WM}/switch-to-workspace-up "['<Primary><Super>Up','<Primary><Super>${up}']"
+    # Move to workspace left
```

Comment:
These keybindings would conflict with half-snapping; I see half-snapping was already disabled in this file, although it's still present in `TESTING.md` and is something we currently test for.